<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Название сорта</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Степень обжарки</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>молотый/в зернах</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Описание вкуса</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Цена</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Объем упаковки</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
